app/authentication/resources.py

```diff
@@ -0,0 +1,40 @@
+from flask import session
+from flask_restful import Resource
+
+from app.users.models import User
+from app.lib.response_util import buildOkResponse, _returnUser
+
+
+class UserLogin(Resource):
+    """Class for logging in a user.
+    """
+    def get(self, userId):
+        user = User.query.get(userId)
+        session['id'] = user.id
```

Comment:
`id` as a key totally works, but who knows we may want to store extra information later, so what do you think about changing the key to `userId` just to be more specific?